app/modules/integrations/integrations_service.py

```
@@ -1016,6 +1023,73 @@ async def _cleanup_jira_webhooks(self, db_integration: Integration) -> None:
+                        )
+                        if success:
+                            deleted_count += 1
+                            logging.info(
+                                f"Deleted Jira webhook {webhook_id} for integration {db_integration.integration_id}"
+                            )
+                        else:
+                            failed_count += 1
+                            logging.warning(
+                                f"Failed to delete Jira webhook {webhook_id}"
+                            )
+                    except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)